src/Application/Controller/Projects.php

```diff
@@ -114,6 +118,25 @@ public function profiles() {
 						);
 					}
 				}
+				
+				if (!empty($values['auto_create']) and
+					is_array($values['auto_create'])) {
+					foreach ($values['auto_create'] as $id => $auto_create) {
+						if (empty($auto_create['1']) && !$auto_create['0'] or
+							$auto_create['1'] === $auto_create['0']) {
+							continue;
+						}
```

Comment:
Is there any reason to update this only when it has changed?

Reply:
Save some unnecessary queries ;)
But I also wasn't able to disable all profiles, since when all checkboxes are unchecked, the $values['auto_create'] key doesn't exist and I didn't want to assume this means all profiles have auto_create = false. I fixed that using a hidden field, but then went ahead and added those conditions to the if-clause.

Reply:
> Save some unnecessary queries ;)

We already have those for priority and this looks very manual, but it's probably fine.

> since when all checkboxes are unchecked, the $values['auto_create'] key doesn't exist and I didn't want to assume this means all profiles have auto_create = false

I don't see any case where this would cause issues?

Reply:
How would solve that? Than I need a disableAutoCreateOnAllProfilesForProject() function in the model or another way to get iterate over all profiles to update them all.
I could use $values['priority'], but this feels messy.

Reply:
> How would you solve that?

If you move these all values to a common array (e.g. `profiles[id][auto_create]`) they should be always filled, because of [Form](https://github.com/jjeising/framework/blob/crs-tools-tracker/Library/Form.php#L158). This currently does not seem to first if the first field is already missing (because of the `index === $maxDepth` condition), maybe I can patch that.